### ISSUE TITLE
LanguageDropdown: sort by default orthography

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#364]
The Language Dropdown should have the language options (but not the "Select a Language" and "Add a Language" options) sorted by the default orthography. Use the `getDefaultLang()` utility method in `/utilities` to sort the data before rendering.

**Comment by vadekh** [comment]
Both here and in issue #161, I think the solution is to sort `this.languages` into a new `sorted` array or something before loading that into the list/dropdown. I will use a helper method that finds the orthography and then orders the languages by that orthography.

**Comment by dwhieb** [comment]
Yeah I agree. Note that `.sort()` sorts an array in place and returns the original array, so you don't have to create a new array. I also just realized that the `utilities/compare.js` utility could be helpful for you with these two issue too. There are code comments at the top of that file explaining how to use it. (Super useful utility - I use it all the time in projects.)
